charts/reform-scan-blob-router/values.yaml

```diff
@@ -32,6 +32,8 @@ java:
     TASK_SCAN_DELAY: "4000" # in millis
 
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 30
+    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
+    STORAGE_BLOB_PUBLIC_KEY: "nonprod_public_key.der"
```

Comment:
How is it going to work? Won't we have the public key in memory somewhere?

Reply:
I added it in #117

Reply:
do we need  special config for Preview?